cypress/integration/ui/about_page.spec.js

```diff
@@ -0,0 +1,25 @@
+describe('about page', () => {
+    before(function () {
```

Comment:
Sometimes you're using full functions like here:
```js
    before(function () {
        cy.visit('/about')
    })
```

and sometimes arrow ones:
```js
    it('loads successfully', () => {
        cy.url().should('include', '/about')
    })
```

Please unif your approach.